test/integration/alerting/test.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+alerting_test() {
+    setup_alerting
+    update_with_file "alerting/install.yaml"
+    install "make"
+    multi_test "alerting/cases.yaml"
+    check_alerting_endpoint_calls
+    uninstall "make"
+    cleanup_alerting
+}
+
+setup_alerting() {
+    # skip if running in CI
+    if [[ "${GITHUB_ACTIONS-}" == "true" ]]; then
```

Comment:
This won't work. That variable is
> The name of the action currently running, or the [id](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsid) of a step. For example, for an action, __repo-owner_name-of-action-repo.

See https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
What you want is 
```suggestion
    if [[ "${CI-}" == "true" ]]; then
```
and it's more generic on top 🎉 
Same below (and I guess in other files too)